lib/libzfs/libzfs_iter.c

```diff
@@ -103,7 +103,14 @@ zfs_do_list_ioctl(zfs_handle_t *zhp, int arg, zfs_cmd_t *zc)
  * Iterate over all child filesystems
  */
 int
-zfs_iter_filesystems(zfs_handle_t *zhp, int flags, zfs_iter_f func, void *data)
+zfs_iter_filesystems(zfs_handle_t *zhp, zfs_iter_f func, void *data)
+{
+	return (zfs_iter_filesystems_v2(zhp, 0, func, data));
```

Comment:
With the `boolean_t` arg added back then this could be changed to:
```
         return (zfs_iter_snapshots_v2(zhp, simple ? ZFS_ITER_SIMPLE : 0,
             func, data, min_txg, max_txg));
````

Reply:
I'm running a test with these changes to verify that everything is working on our end.

Reply:
Good spot, all taken as suggested.